packages/block-serialization-default-parser/parser.php

```diff
@@ -1,4 +1,9 @@
 <?php
+/**
+ * Block Serialization Parser
+ *
+ * @package WordPress
```

Comment:
One thing I just remembered I was going to ask...I think this should probably be `@package gutenberg` instead of `WordPress`.

Reply:
It is copied over to core causing issues there when listed as gutenberg. I think that’s why PHP files which are shared need be attributed to WordPress. The same issue was with transportations.

Reply:
The issues in core were related to white space that got auto-fixed by `grunt precommit`, so this portion won’t really matter for that, but I had to assign the @package here to get it to pass the sniff. 

First thought was WordPress because it’s copied over into `wp-includes`, so that’s what I set, but then thought Gutenberg might make more sense because that is the actual origin of the file.

Reply:
Yeah it needs to be `@package WordPress` because it's copied to core.